feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/FloorMapScreen.kt

```diff
@@ -138,6 +142,18 @@ private fun FloorMapScreen(
                             .padding(bottom = 56.dp),
                     )
                 }
+                Spacer(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .height(96.dp)
+                        .align(Alignment.BottomCenter)
+                        .background(
+                            brush = Brush.verticalGradient(
```

Comment:
There is a `Brush.verticalGradient` already here. Could we use it?
This might be not an easy problem 👀 
https://github.com/DroidKaigi/conference-app-2023/pull/690/files#diff-631fb12247c515279975a2569c2ac6255f80753353c8b7a97b071696c2b8a6c1R137

Reply:
Thanks for pointing that out!
The `fadingEdge` was not visible, so I fixed it.

In the previous order of modifier, blur was drawn first and then paddig was applied.The blur was not visible because it was hidden by the padding.
So, I set the padding first and then draw the bluer in the display area, the code is now as intended!


Before | After(Light Theme)|After(Dark Theme)|
:--: | :--:| :--:
<img src="https://github.com/DroidKaigi/conference-app-2023/assets/64339483/3731c490-20b7-4f12-98ab-21ea9ab4963f" width="300" /> |<img src="https://github.com/DroidKaigi/conference-app-2023/assets/64339483/0b37c045-5e17-436d-8870-1b0520838fe4" width="300" /> |<img src="https://github.com/DroidKaigi/conference-app-2023/assets/64339483/628773bf-44dc-4e24-b8fd-867aa3205a87" width="300" />